metricbeat/module/ceph/osd_df/_meta/fields.yml

```diff
@@ -0,0 +1,41 @@
+- name: osd_df
+  type: group
```

Comment:
It's a new feature I just added recently. Could you add `  release: experimental
` here? Like this you don't have to add it manually to the docs.

Reply:
Ok, I will add it.